genesis-builder/src/lib.rs

```diff
@@ -187,6 +196,46 @@ impl GenesisBuilder {
                 .expect("Failed to store account");
         }
 
+        debug!("Vesting contracts");
```

Comment:
Is this needed?

Reply:
No, but thats to be consistent with the other logs that were already in the file